pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -545,6 +559,20 @@ func (a Adapter) createOrUpdateComponent(componentExists bool, ei envinfo.EnvSpe
 	return nil
 }
 
+// getDeploymentObjectMeta generates a ObjectMeta object for the given deployment's name and labels
```

Comment:
name of function and comment do not match

Reply:
Fixed